Libraries/src/Amazon.Lambda.RuntimeSupport/ExceptionHandling/StackFrameInfo.cs

```diff
@@ -26,24 +27,40 @@ public StackFrameInfo(string path, int line, string label)
             Label = label;
         }
 
+#if NET8_0_OR_GREATER
+        [System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026",
+            Justification = "Constructor has defensive code in place in case the method for the stack frame has been trimmed.")]
+#endif
         public StackFrameInfo(StackFrame stackFrame)
         {
             Path = stackFrame.GetFileName();
             Line = stackFrame.GetFileLineNumber();
 
-            var method = stackFrame.GetMethod();
-            if (method != null)
+            try
             {
-                var methodTypeName = method.DeclaringType?.Name;
-                if (methodTypeName == null)
-                {
-                    Label = method.Name;
-                }
-                else
+                var method = stackFrame.GetMethod();
```

Comment:
This `GetMethod` call is marked as `RequiresUnreferencedCode`. The code is required so I just made the code more defensive in case something gets trimmed. Since we only really need the that is unlikely to be metadata trimmed.